### ISSUE TITLE
(67) dde de changement de nom d'un SCoT

### ISSUE THREAD

**Issue by CeliaVermicelli** [MTES-MCT/Docurba#645]
Page de l'EPCI : https://docurba.beta.gouv.fr/ddt/68/collectivites/200066058/epci

Page de la FRP : https://docurba.beta.gouv.fr/frise/ed9fa567-04bc-47fa-9eec-1f9e5bbe52bb ( FYI le nom du SCoT dans la FRP est le bon)

-> Le nom du ScoT depuis la page de l'EPCI devrait aussi être "SCoT de CA Saint Louis Agglomération" : 

<img width="1309" alt="Capture d’écran 2024-05-30 à 3 51 33 PM" src="https://github.com/MTES-MCT/Docurba/assets/111045572/4a21b09e-0230-4e7a-9c1a-2bece8ed6048">

**Comment by HermanceGauth** [comment]
@UngererFabien non résolu avec maj du référentiel en dev

**Comment by rik** [comment]
Le titre a été harmonisé en utilisant le nom présent dans la base de données : SCOT DU PAYS DE SAINT-LOUIS ET DES TROIS FRONTIERES.

Passage à @CeliaVermicelli pour vérification.

**Comment by CeliaVermicelli** [comment]
Effectivement, SCOT DU PAYS DE SAINT-LOUIS ET DES TROIS FRONTIERES est affiché sur la FRP et la page de la CT 

### ❌ Mais cela devrait être "SCoT de Saint-Louis Agglomération"
**ci-après le contexte de la DDT si utile.**

![Capture d’écran 2025-01-15 à 4 49 33 PM](https://github.com/user-attachments/assets/76e175ca-c2ab-4bee-a5c2-23e4df45f803)
![Capture d’écran 2025-01-15 à 4 49 28 PM](https://github.com/user-attachments/assets/5412260f-0366-49c6-b4f8-4c967638c858)


> Ce SCoT a changé de nom à plusieurs reprises.
> Aujourd'hui, son intitulé officiel est "SCoT de Saint-Louis Agglomération".
> [...]
> Le 30/06/2022, le SCoT révisé a été approuvé. La délibération d'approbation a décidé explicitement de le nommer "SCoT de Saint-Louis Agglomération" (SCoT SLA) (cf pj p4).

**Comment by CeliaVermicelli** [comment]
@HermanceGauth @rik 
Suite à mon dernier commentaire et après double vérification avec la DDT : que fait-on de ces cas où le référentiel n'est "pas conforme" au terrain ?
